examples/README.md

```diff
@@ -255,6 +255,7 @@ Example | Description
 --- | ---
 [2D top-down camera](../examples/camera/2d_top_down_camera.rs) | A 2D top-down camera smoothly following player movements
 [First person view model](../examples/camera/first_person_view_model.rs) | A first-person camera that uses a world model and a view model with different field of views (FOV)
+[Projection Zoom](../examples/camera/projection_zoom.rs) | Shows how to zoom and orbit orthographic and perspective projection cameras.
```

Comment:
I quite like the orbit showcase. I don't think we had that before :) 
Maybe rename the file to `orbit_and_projection_zoom`, with the example name changing accordingly?